### PR TITLE
wr-120: Update app logo and icon

### PR DESCRIPTION
**Pull request by IrynaK-a**
Auth:
image

App:
image

Header:
image

### CODE REVIEW

File: frontend/src/assets/img/icons/writorium-auth-page-logo.svg
Comment (andanf-e) [inline frontend/src/assets/img/icons/writorium-auth-page-logo.svg:1]: Let's add it to icon + I guess we may use single one and just change sizing\color

File: frontend/src/assets/img/icons/writorium-header-logo.svg
Comment (drMangust) [inline frontend/src/assets/img/icons/writorium-header-logo.svg:1]: It's the same logo as [auth-page](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/127/files#diff-21db1c98fb4da97e17f667d067cef338829eeb788a76590f6947920a1b2077dd), but the color is different, can you leave one and change the color when you need to? But it up to you... 

remove fill from path:
image
and add to svg
image

We discussed this here  https://bsa2023.slack.com/archives/C05N8A6M9T5/p1693580714202949

Comment (drMangust) [reply frontend/src/assets/img/icons/writorium-header-logo.svg:1]: I see Andrew has already written about it

Comment (IrynaK-a) [reply frontend/src/assets/img/icons/writorium-header-logo.svg:1]: Yes, I got it, thanks.

File: frontend/src/libs/components/header/header.tsx [hunk 1]
```diff
@@ -13,8 +13,8 @@ const Header: React.FC<Properties> = ({ user }) => (
   <>
     {user && (
       <header className={styles.header}>
-        <Link to={AppRoute.ROOT} className={styles.logo}>
-          WRITORIUM
+        <Link to={AppRoute.ROOT}>
+          <Icon iconName={'writoriumLogo'} className={styles.logo} />
```
Comment (artemmatiushenko1) [inline frontend/src/libs/components/header/header.tsx]: `iconName='writoriumLogo'`

File: frontend/src/pages/auth/components/auth-layout/styles.module.scss [hunk 2]
```diff
@@ -71,9 +71,7 @@
 }
 
 .heroTitle {
+  width: 615px;
```
Comment (artemmatiushenko1) [inline frontend/src/pages/auth/components/auth-layout/styles.module.scss]: Let’s better specify a `max-width: 615px` and `width: 100%` to make it responsive

File: frontend/src/pages/auth/components/auth-layout/auth-layout.tsx [hunk 3]
```diff
@@ -33,7 +35,7 @@ const AuthLayout: React.FC<Properties> = ({ children }) => {
       </div>
       <div className={styles.hero}>
         {renderFloatingLetters(HERO_FLOATING_LETTERS)}
-        <h1 className={styles.heroTitle}>WRITORIUM</h1>
+        <Icon iconName={'writoriumLogo'} className={styles.heroTitle} />
```
Comment (artemmatiushenko1) [inline frontend/src/pages/auth/components/auth-layout/auth-layout.tsx]: `iconName='writoriumLogo'`

**Review by andanf-e — DISMISSED** [review]

**Review by andanf-e — DISMISSED** [review]

**Review by mhrechkin — DISMISSED** [review]

**Review by NazarKovtun1 — DISMISSED** [review]

**Review by NazarKovtun1 — DISMISSED** [review]

**Review by andanf-e — DISMISSED** [review]

**Review by andanf-e — APPROVED** [review]

**Review by NazarKovtun1 — APPROVED** [review]

**Pull request merged**